<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise - Restoration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="images_layout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="original_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Original</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="noisyRestored_layout">
        <item>
         <widget class="QLabel" name="noisy_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Noisy</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="restored_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Restored</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Noise</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="gaussian_radio">
             <property name="text">
              <string>Gaussian</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="gamma_radio">
             <property name="text">
              <string>Gamma</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="salt_pepper_radio">
             <property name="text">
              <string>Salt and Pepper</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="exponential_radio">
             <property name="text">
              <string>Exponential</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rayleigh_radio">
             <property name="text">
              <string>Rayleigh</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="uniform_radio">
             <property name="text">
              <string>Uniform</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">noise_radio_group</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>First property:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Second property</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Filter</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButton_9">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioButton_10">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioButton_8">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton_7">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButton_11">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="radioButton_12">
             <property name="text">
              <string>RadioButton</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">filter_radio_group</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>First property:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Second property:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Third property:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Load image</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Browse folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="images_list">
           <property name="defaultDropAction">
            <enum>Qt::IgnoreAction</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="isWrapping" stdset="0">
            <bool>true</bool>
           </property>
           <property name="gridSize">
            <size>
             <width>225</width>
             <height>100</height>
            </size>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="noise_radio_group"/>
  <buttongroup name="filter_radio_group"/>
 </buttongroups>
</ui>
